### DISCUSSION TITLE
No GPS present

### DISCUSSION THREAD

**Discussion by Th3H0ff** [meshtastic/Discussions#91] [General]
Hi all
I've been banging my head for a few hours on this and I need some suggestion on where to troubleshoot.
Hardware: Lilygo T3 V1.6.1
Version: 2.5.11.8e2a3e5
GPS: GT-U7
Pins. T3-GPS
3V-VCC
GND-GND
GPIO12-TX

In app config.
![Screenshot_20241124_133511_Meshtastic](https://github.com/user-attachments/assets/3abcd00b-1a0e-4f8a-be10-f86e2aabbf7d)

The GPS LED is solid red after reboot and then starts blinking which I read suggests in has a GPS fix. Lilygo screen still says No GPS.

**Comment by fifieldt** [comment]
Hi @Th3H0ff , can you try the latest alpha for me?

and/Or, can you jump on https://flasher.meshtastic.org and use the "serial console" around the time of startup and paste your logs here?

**Reply by Th3H0ff** [reply]
I got some input from the discord channel. Despite several guides only using one pin for RX it seems I needed both RX and TX connected. Once I did that the GPS came online.

**Reply by fifieldt** [reply]
Glad to hear it!
